ops/k8s/deployments/api-deployment.yaml

```diff
@@ -14,6 +14,7 @@ spec:
         app: data-repo
         component: data-repo-api
     spec:
+      serviceAccountName: jade-sa
```

Comment:
Is that a specific service account? Or just a name to call the actual service account?

Reply:
These are not equivalent to GCP SAs -- it is a name for the k8s specific SA.